src/main/kotlin/com/jaredsburrows/spoon/SpoonPlugin.kt

```diff
@@ -34,7 +34,16 @@ class SpoonPlugin : Plugin<Project> {
           description = "Run instrumentation tests for '${variant.name}' variant."
           group = "Verification"
           outputs.upToDateWhen { false }
-          dependsOn(variant.testedVariant.assemble, variant.assemble)
+          variant.testedVariant.assembleProvider.configure { provider ->
```

Comment:
I guess we can just stick to 
https://github.com/jaredsburrows/gradle-spoon-plugin/pull/68/files#diff-8ced7b9d1d36deac751b5d9ddc0ce435R37
my PR is almost ready apart from fact that it passes locally not on TravisCI :/